pkg/controller/planexecution/planexecution_controller.go

```diff
@@ -381,14 +381,13 @@ func (r *ReconcilePlanExecution) Reconcile(request reconcile.Request) (reconcile
 						NamePrefix: instance.Name + "-",
 						Namespace:  instance.Namespace,
 						CommonLabels: map[string]string{
-							"heritage":      "kudo",
-							"app":           frameworkVersion.Spec.Framework.Name,
-							"version":       frameworkVersion.Spec.Version,
-							"instance":      instance.Name,
-							"planexecution": planExecution.Name,
```

Comment:
This would change when an update happened and it would cause errors.  We might still have issues when we update with a different plan: